### DISCUSSION TITLE
Are there actually any realistic plans of getting this to project to .NET8+?

### DISCUSSION THREAD

**Discussion by danielwertheim** [AutoFixture/AutoFixture#1441] [General]
I get that this is an open source project and that we all can invest our time in it and help out. How-ever, do the maintainer(s) have any realistic plan and active goals of e.g lifting this project to support e.g. .NET8+ with proper dependencies etc so that we e.g. get rid of vulnerability warnings from transient dependencies etc.?

**Comment by aivascu** [comment]
@danielwertheim yes there are plans to move to .NET 8. In fact I have been working on this for the past week.
Unfortunately this not as easy as adding net8.0 to the target frameworks.

**Comment by danielwertheim** [comment]
Great news! Also, I do get that this is open source and that there isn't endless amount of resources etc that one person can put in.

Why I'm asking is because there are still "issues" (never liked that naming from GitHub) up for .NET5, .NET6 which also mentions .NET7 etc.  Hence why I'm asking if there are any realistic plans for .NET8 in the near future. And I do get that it's not as easy as just adding multitarget support and conditional references and be done with it. But if .NET8 "plans" will pan out in the same manner as those "issues" (.NET5, 6), then it means: "Yes, the intentions are there but the resources are not" which means it's up to us to either "dig in" or "move on" or be quiet so to speak.

**Reply by aivascu** [reply]
I get your frustration. I think an updated issue for .NET is in order just to keep everyone informed of what the plans are.
